### PR TITLE
updating guide to say that pro is now free

### PR DESCRIPTION
**Pull request by dennisnewel**
Not sure about the bolding of the text in the bracket?

### CODE REVIEW

File: _pro/getting-started/getting-started-part-three.md [hunk 1]
```diff
@@ -29,13 +29,12 @@ Now that you have a working CI process with a successful test, the next step is
 
 ### A Codeship Account With Access To Code Pro
```
Comment (ethangj) [inline _pro/getting-started/getting-started-part-three.md]: @dennisnewel can you also fix the headline here? "Code Pro" 😛

**Comment by dennisnewel** [comment]
I can fix that (still have the doc open)

**Comment by ethangj** [comment]
Lgtm

**Comment by dennisnewel** [comment]
@ethangj is there a way i can 'accept' your review? it looks like the changes are still blocked?

**Pull request merged**